app/src/main/java/io/horizontalsystems/bitcoinkit/demo/MainViewModel.kt

```diff
@@ -66,9 +67,10 @@ class MainViewModel : ViewModel(), BitcoinKit.Listener {
                 386526600, 187228578,
                 "00000000000000000005cff1d36f64f1ceef5d8a0da26ebbd671f1d1a9a1cff6".toReversedByteArray()), 750792)
 
+        val authKey = ""
```

Comment:
Seems weird to make an empty val can just pass "" to the constructor AuthKey is always empty?

Reply:
See above comment